app/src/format.py

```diff
@@ -43,6 +43,8 @@ def format_accordion_body(self, citation_body: str) -> str:
 def to_html(text: str) -> str:
     # markdown expects '\n' before the start of a list
     corrected_text = re.sub(r"^([\-\+\*]) ", "\n- ", text, flags=re.MULTILINE, count=1)
+    # markdown expect 4 spaces before a nested list item; LLMs often use 3 spaces
+    corrected_text = re.sub(r"^   ([\-\+\*]) ", "    - ", corrected_text, flags=re.MULTILINE)
```

Comment:
`https://github.com/Python-Markdown/markdown/issues/3` (x-ref without causing a backlink to the issue since it has backlinked numerous times.